config/locales/en/placements/schools/placements.yml

```diff
@@ -8,8 +8,10 @@ en:
               invalid: Select a phase
             mentor_ids:
               invalid: Select a mentor or not yet known
-            subject_ids:
+            subject:
               invalid: Select a subject
+            additional_subject_ids:
+              invalid: Select an additional subject
```

Comment:
I'm not _entirely sure_ if "additional subject" is the right term to use here. While we've modelled it internally as "additional subjects", the prototype & UI doesn't ever call them that.

But I think this should be fine for now. I'd rather not over-complicate things or pre-empt the findings of the upcoming content review.